components/style/properties/properties.mako.rs

```diff
@@ -2300,3 +2315,21 @@ macro_rules! longhand_properties_idents {
         }
     }
 }
+
+// If you update this, please also update the `CascadeInfo::on_cascade_boxed_property`.
+<%def name="declare_has_viewport_percentage_inner(variable, boxed)">
```

Comment:
I created this as mako function and used in two places. At least this will avoid some update problems. But we still need to update `on_cascade_boxed_property` in case of update